examples/pes/instanton/ch4h-cbe-rate/instanton/20/inst.instanton_forces_0.xyz

```diff
@@ -0,0 +1,160 @@
+H -0.00022441272173878699 0.00047954236137570655 -0.000235235986245641
+H 0.0027857110229678734 0.0032713197706257797 0.0027936063740074952
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       15
+H 0.0029727852013383678 0.0034805329951481663 0.002967803730546947
+C -0.0065007241478731706 -0.007575057386910443 -0.006443031025371581
+H -0.00021833880259336522 -0.000218606416302336 0.0005377863210309641
+H 0.0005678091274941011 -0.00023273504234921913 -0.00023783625069384581
+H -0.00026056176105626605 0.0005078040488104563 -0.0002729758363795298
+H 0.003439030382690333 0.0040380618460122975 0.003448253038662585
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       16
+H 0.0027232277655109267 0.003187433345042123 0.0027174433203658306
+C -0.006751834641383069 -0.007863330636626031 -0.006686332187921805
+H -0.00024446306090553094 -0.0002471147508842364 0.0005614649467577948
+H 0.0005952195902825252 -0.0002634909401777463 -0.0002669614140859977
+H -0.00028993880629712976 0.0005279188031170889 -0.0003036572271408744
+H 0.003967789174996739 0.004658584190631032 0.003978042550922822
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       17
+H 0.0025177789519936766 0.0029461820261200695 0.002511391827830778
+C -0.0069201691488274255 -0.008055838579545949 -0.006848480760268671
+H -0.00026420580723218734 -0.0002687666755818441 0.0005776909062227276
+H 0.000614402984044915 -0.0002869329662757991 -0.00028908147542416884
+H -0.00031209015927124995 0.0005414900250411137 -0.0003268023796021424
+H 0.004364283179292272 0.005123866148037948 0.004375281881241477
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       18
+H 0.0023724504694122572 0.0027755488019032266 0.0023656647973879785
+C -0.007023739068134915 -0.008173958876422915 -0.006947829700454377
+H -0.0002773346274853594 -0.00028321334166747647 0.0005877645981033908
+H 0.0006264933349875434 -0.0003025991790472915 -0.0003038314988490498
+H -0.0003267960624331323 0.0005498122346292433 -0.0003421755656596303
+H 0.004628925964755837 0.005434410393911903 0.004640407391676149
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       19
+H 0.00229626408443373 0.002686105171623154 0.0022892805817686934
+C -0.007073577290572785 -0.00823075345657287 -0.006995578460955442
+H -0.0002838161816676177 -0.00029035772119989645 0.0005925823221009097
+H 0.0006323069401403814 -0.0003103414025762419 -0.0003111145285839001
+H -0.00033404736199393875 0.0005537652114639968 -0.0003497600653545873
+H 0.004762869793006885 0.005591582169506281 0.004774590162126557
```

Comment:
Do we need this file in the main repo? What is its purpose?

Reply:
No, we don't. I will delete it.